ipmininet/router/config/exabgp.py

```diff
@@ -246,16 +280,33 @@ def dry_run(self) -> str:
 
     def set_defaults(self, defaults):
         """
-        :param env: a dictionary of all the environment variables that configure ExaBGP
+        :param env: a dictionary of all the environment variables that configure ExaBGP.
                     Type "exabgp --help" to take a look on every environment variable.
                     env.tcp.delay is set by default to 2 min as FRRouting BGPD daemon
                     seems to reject routes if ExaBGP injects routes to early.
-               address_families: the routes to inject for both IPv4 and IPv6 unicast
-                                 AFI.
-               passive: Tells to ExaBGP to not send active BGP Open messages. The daemon
-                        waits until the remote peer send first the Open message to start
+                    Each environment variable is either a string or an int.
+
+                    The following environment variable are set :
+
+                    * daemon.user = 'root'
+                    * daemon.drop = 'false'
+                    * daemon.daemonize = 'false'
+                    * daemon.pid = <default configuration folder /tmp/exabgp_<node>.pid>
+
+                    * log.level = 'CRIT'
+                    * log.destination = <default configuration folder /tmp/exabgp_<node>.log>
+                    * log.reactor = 'false'
+                    * log.processes = false'
+                    * log.network = 'false'
+
+                    * api.cli = 'false'
+
+                    * tcp.delay = 2 # waits at most 2 minutes
+        :param address_families: the routes to inject for both IPv4 and IPv6 unicast AFI.
+        :param passive: Tells to ExaBGP to not send active BGP Open messages. The daemon
+                        waits until the remote peer sends first the Open message to start
                         the BGP session. Its default value is set to True.
-        :return:
+        :return: The default configuration of this ExaBGP daemon
```

Comment:
It does not really returns, only modifies

Reply:
Changed: 4af2eb9c67192d505d51b10d1ca6f22936272221